tests/testthat/test-hybrid.R

```diff
@@ -1,6 +1,10 @@
 patrick::with_parameters_test_that("basic hybrid test", {
   if(ci_skip == TRUE) skip_on_ci()
 
+  if (skip) {
```

Comment:
I'd consider using a less generic varname. E.g., `fulldata_testcase`.

Reply:
done in [be14dc8](https://github.com/RECETOX/recetox-aplcms/pull/177/commits/be14dc89f1cab957c595bded05e21280093f11a9)